frontend/src/views/Layout/DashboardLayout_gdd.vue

```diff
@@ -99,14 +101,17 @@ export default {
       this.$router.push('/login')
     },
     async updateTransactions() {
+      this.pending = true
       const result = await communityAPI.transactions(this.$store.state.sessionId)
       if (result.success) {
         this.GdtBalance = Number(result.result.data.gdtSum)
         this.transactions = result.result.data.transactions
         this.balance = Number(result.result.data.decay)
         this.bookedBalance = Number(result.result.data.balance)
         this.transactionCount = result.result.data.count
+        this.pending = false
       } else {
+        this.pending = false
```

Comment:
May be better to keep pending true here. So when the API call fails, no balance is shown (instead of 0)

Reply:
That would be a bit dirty I guess but totally acceptable. Do it!